src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -238,7 +237,7 @@ private void setSelectedMap(MapBean newValue) {
         new BackgroundSize(BackgroundSize.AUTO, BackgroundSize.AUTO, false, false, true, false))));
 
     MapSize mapSize = newValue.getSize();
-    mapSizeLabel.setText(i18n.get("mapPreview.size", mapSize.getWidthInPixels(), mapSize.getHeightInPixels()));
+    mapSizeLabel.setText(i18n.get("mapPreview.size", mapSize.getWidthInKm(), mapSize.getHeightInKm()));
```

Comment:
Do we wanna include the term km in mapOreview.size?

Reply:
I would say yes. Regarding size in km there is:
* mapPreview.size
* mapVault.upload.sizeFormat
* map.size (unsure, unused?)

In general I suggest to include the unit in the message key if the unit is used in the translation.